client/src/components/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import CreateRecipe from './CreateRecipe';
+
+class App extends React.Component {
```

Comment:
If it's going to be stateless,  functional component would be preferred.

Reply:
Going to be stateful in the next PR.